specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -1358,6 +1358,323 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/kustoPools/{kustoPoolName}/attachedDatabaseConfigurations": {
+      "get": {
+        "tags": [
+          "AttachedDatabaseConfigurations"
+        ],
+        "description": "Returns the list of attached database configurations of the given Kusto pool.",
+        "operationId": "AttachedDatabaseConfigurations_ListByKustoPool",
+        "x-ms-examples": {
+          "KustoPoolAttachedDatabaseConfigurationsListByKustoPool": {
+            "$ref": "./examples/KustoPoolAttachedDatabaseConfigurationsListByKustoPool.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/kustoPoolNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of attached database configurations.",
+            "schema": {
+              "$ref": "#/definitions/AttachedDatabaseConfigurationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/kustoPools/{kustoPoolName}/attachedDatabaseConfigurations/{attachedDatabaseConfigurationName}": {
+      "get": {
+        "tags": [
+          "AttachedDatabaseConfigurations"
+        ],
+        "description": "Returns an attached database configuration.",
+        "operationId": "AttachedDatabaseConfigurations_Get",
+        "x-ms-examples": {
+          "KustoPoolAttachedDatabaseConfigurationsGet": {
+            "$ref": "./examples/KustoPoolAttachedDatabaseConfigurationsGet.json"
+          }
+        },
+        "parameters": [
+          {
```

Comment:
parameters should appear in the order they appear in the uri:
- by path segments, in order
- query parameters
- body parameters